src/lesson_12/hometask_12/Student.java

```diff
@@ -0,0 +1,70 @@
+package lesson_12.hometask_12;
+
+import java.io.File;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Student {
+    private String name;
+    ArrayList<File> fileList = new ArrayList<>();
+    private int score;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getScore() {
+        return score;
+    }
+
+    public void setScore(int score) {
+        this.score = score;
+    }
+
+    public Student(String name, ArrayList<String> fileList, int score) {
+        this.name = name;
+        for (String fileName : fileList) {
+            try {
+                this.fileList.add(new File(fileName));
+            } catch(NullPointerException e) {
+                this.fileList.add(null);
+            }
+        }
+        this.score = score;
+    }
+
+    private int checkFileExtension(String fileName) throws FileException{
```

Comment:
method must return the numbers of the points for this specific file not the overall score